src/detections/rule/matchers.rs

```diff
@@ -2395,4 +2408,80 @@ mod tests {
 
         check_select(rule_str, record_json_str, false);
     }
+
+    #[test]
```

Comment:
Can you please create a test for a test case where there is no IpAddress|cidr in the rule, but there is no IpAddress field in the record?
(This fix should solve the problem with https://github.com/Yamato-Security/hayabusa/pull/961#issuecomment-147097673 test case. The expectation in that test case is that the number of cases to be detected is zero.)

Reply:
I will also add test case and check it!